Packs/SymantecEDR/Integrations/SymantecEDR/SymantecEDR.py

```diff
@@ -0,0 +1,2434 @@
+                json_data=params if method == 'POST' else {},
+                params=params if method == 'GET' else {},
+                resp_type='response',
+                allow_redirects=False
+            )
+            response.raise_for_status()
+        except requests.exceptions.HTTPError as err:
+            if HTTP_ERRORS.get(err.response.status_code) is not None:
+                # if it is a known http error - get the message form the preset messages
+                err_response = f'Failed to execute. {response.json().get("error")},{response.json().get("message")}'
+                msg = f'{HTTP_ERRORS.get(err.response.status_code)}.\n{err_response}'
+                raise DemistoException(msg, res=response)
+            else:
+                # if it is unknown error - get the message from the error itself
+                return_error(f'Failed to execute. Error: {str(err)}')
+        except Exception as e:
+            return_error(f'Failed to execute. Error: {str(e)}')
+
+        return response.json()
+
+    def query_patch_api(self, endpoint: str, payload: list) -> dict:
+        """
+        Call the PATCH api to add/modify or update to the endpoint
+        Args:
+            endpoint (str): Symantec EDR endpoint resources operation add, update, delete
+            payload (List): request body
+        Returns:
+            return response status code
+        """
+
+        result: dict = {}
+        access_token = self.get_access_token()
+        url_path = f'{self._base_url}{endpoint}'
+
+        headers = {
+            'Content-Type': 'application/json',
+            'Authorization': f'Bearer {access_token}'
+        }
+
+        response = self._http_request(
+            method="PATCH",
+            headers=headers,
+            data=json.dumps(payload),
+            full_url=url_path,
+            resp_type="response",
+            return_empty_response=True,
+        )
+
+        if response.status_code == 204:
+            result['status'] = response.status_code
+            result['message'] = 'Success'
+
+        if response.status_code >= 400:
+            error_message = f'{response.json().get("error")}, {response.json().get("message")}'
+            raise DemistoException(error_message, res=response)
+
+        return result
+
+
+''' HELPER FUNCTIONS '''
+
+
+def handle_errors(response: requests.Response) -> None:
+    """Handle http errors."""
+    status = response.status_code
+    if status in HTTP_ERRORS:
+        raise DemistoException(f'{HTTP_ERRORS[status]}, {response.json().get("error_description")}', res=response)
+    else:
+        response.raise_for_status()
+
+
+def http_request_error_handler(response: requests.Response):
+    """
+    Error Handler for Symantec EDR on-premise
+    Args:
+        response (response): Symantec EDR on-premise response
+    Raise:
+         DemistoException
+    """
+    if response.status_code >= 400:
```

Comment:
this is unnecessary, given we use `raise_for_status`

Reply:
Okay. Removed now.